README.md

```diff
@@ -1,40 +1,67 @@
 # evmosjs
 
-JS and TS libs for Evmos
+JS and TS libs for Evmos.
 
-## Example
+## Installation
 
-### Get account information
+evmosjs uses [buf.build](https://buf.build/) to manage Protobuf dependencies. To install evmosjs packages in your project,
+follow the instructions corresponding to your package manager.
 
-Get the account number, sequence and pubkey from an address.
-NOTE: if the address had not sent any transaction to the blockchain, the pubkey value are going to be empty.
+### NPM
+
+Add the following line to an `.npmrc` file in your project root:
+
+```ini
```

Comment:
is `ini` a valid option?

Reply:
Yes, that's the format of the `.npmrc` file:
https://docs.npmjs.com/cli/v9/configuring-npm/npmrc#global-config-file

GitHub uses Linguist to parse MarkDown:
https://github.com/github/linguist

Which supports Ini:
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

Most text editors should also support it by way of Highlight.js:
https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md